floris/uncertain_floris_model.py

```diff
@@ -29,20 +29,27 @@ class UncertainFlorisModel(LoggingManager):
                 - **farm**: See `floris.simulation.farm.Farm` for more details.
                 - **turbine**: See `floris.simulation.turbine.Turbine` for more details.
                 - **wake**: See `floris.simulation.wake.WakeManager` for more details.
-                - **logging**: See `floris.core.Core` for more details.
-        wd_resolution (float, optional): The resolution of wind direction, in degrees.
-            Defaults to 1.0.
+                - **logging**: See `floris.simulation.core.Core` for more details.
+        wd_resolution (float, optional): The resolution of wind direction for generating
+            gaussian blends, in degrees.  Defaults to 1.0.
         ws_resolution (float, optional): The resolution of wind speed, in m/s. Defaults to 1.0.
-        ti_resolution (float, optional): The resolution of turbulence intensity. Defaults to 0.01.
-        yaw_resolution (float, optional): The resolution of yaw angle, in degrees. Defaults to 1.0.
+        ti_resolution (float, optional): The resolution of turbulence intensity.
+            Defaults to 0.01.
+        yaw_resolution (float, optional): The resolution of yaw angle, in degrees.
+            Defaults to 1.0.
         power_setpoint_resolution (int, optional): The resolution of power setpoints, in kW.
             Defaults to 100.
         wd_std (float, optional): The standard deviation of wind direction. Defaults to 3.0.
         wd_sample_points (list[float], optional): The sample points for wind direction.
             If not provided, defaults to [-2 * wd_std, -1 * wd_std, 0, wd_std, 2 * wd_std].
+        fix_yaw_to_nominal_direction (bool, optional): Fix the yaw angle to the nominal
+            direction?   When False,
+            the yaw angle is the same across the sampled wind directions.  When True, the yaw
+            angle is fixed to the nominal wind direction and so of offset by the negative of
+            wind direction offset. Defaults to False.
```

Comment:
This description is a little confusing to me. Could this say "When False, the yaw misalignment is the same across the sampled wind directions. When True, the turbine orientation is fixed to the nominal wind direction such that the yaw misalignment changes depending on the sampled wind direction."

Reply:
Done!